server/test/clients.json

```diff
@@ -0,0 +1,112 @@
+            "ReferredBy": "Larry",
+            "BirthDate": "8/21/1987",
+            "MobilePhone": "810-292-9388"
+          },
+          {
+            "FirstName": "Art",
+            "LastName": "Venere",
+            "AddressLine1": "8 W Cerritos Ave #54",
+            "State": "NJ",
+            "City": "Bridgeport",
+            "PostalCode": "8014",
+            "Email": "art@venere.org",
+            "ReferredBy": "Julie",
+            "BirthDate": "5/14/1999",
+            "MobilePhone": "856-636-8749"
+          },
+          {
+            "FirstName": "Lenna",
+            "LastName": "Paprocki",
+            "AddressLine1": "639 Main St",
+            "State": "AK",
+            "City": "Anchorage",
+            "PostalCode": "99501",
+            "Email": "lpaprocki@hotmail.com",
+            "ReferredBy": "Monica",
+            "BirthDate": "12/1/1980",
+            "MobilePhone": "907-385-4412"
+          },
+          {
+            "FirstName": "Donette",
+            "LastName": "Foller",
+            "AddressLine1": "34 Center St",
+            "State": "OH",
+            "City": "Hamilton",
+            "PostalCode": "45011",
+            "Email": "donette.foller@cox.net",
+            "ReferredBy": "James",
+            "BirthDate": "3/31/1967",
+            "MobilePhone": "513-570-1893"
+          },
+          {
+            "FirstName": "Simona",
+            "LastName": "Morasca",
+            "AddressLine1": "3 Mcauley Dr",
+            "State": "OH",
+            "City": "Ashland",
+            "PostalCode": "44805",
+            "Email": "simona@morasca.com",
+            "ReferredBy": "Natalie",
+            "BirthDate": "2/21/1997",
+            "MobilePhone": "419-503-2484"
+          },
+          {
+            "FirstName": "Mitsue",
+            "LastName": "Tollner",
+            "AddressLine1": "7 Eads St",
+            "State": "IL",
+            "City": "Chicago",
+            "PostalCode": "60632",
+            "Email": "mitsue_tollner@yahoo.com",
+            "ReferredBy": "Sarah",
+            "BirthDate": "4/7/2000",
+            "MobilePhone": "773-573-6914"
+          },
+          {
+            "FirstName": "Leota",
+            "LastName": "Dilliard",
+            "AddressLine1": "7 W Jackson Blvd",
+            "State": "CA",
+            "City": "San Jose",
+            "PostalCode": "95111",
+            "Email": "leota@hotmail.com",
+            "ReferredBy": "Josh",
+            "BirthDate": "10/10/2001",
+            "MobilePhone": "408-752-3500"
+          },
+          {
+            "FirstName": "Sage",
+            "LastName": "Wieser",
+            "AddressLine1": "5 Boston Ave #88",
+            "State": "SD",
+            "City": "Sioux Falls",
+            "PostalCode": "57105",
+            "Email": "sage_wieser@cox.net",
+            "ReferredBy": "Chris",
+            "BirthDate": "11/20/1995",
+            "MobilePhone": "605-414-2147"
+          }
+    ]
+}
```

Comment:
More a nit pick than a real problem but it is a best practice to add newlines to the ends of files when working with git.